dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -507,9 +507,11 @@ DomainParticipantImpl::create_topic_i(
       return new_topic;
 
     } else {
-      ACE_ERROR((LM_ERROR,
-                 ACE_TEXT("(%P|%t) ERROR: DomainParticipantImpl::create_topic, ")
-                 ACE_TEXT("assert_topic failed with return value %d.\n"), status));
+      if (DCPS_debug_level >= 1) {
```

Comment:
There are a number of places where the error was considered severe enough to log unconditionally.  Looks like it has been this way for a while.  I'm not opposed to changing it, was there a particular test that prompted this?

Reply:
I noticed it when testing AXCIOMA with OpenDDS integration but also the CIAO with OpenDDS integration shows the same, see http://scoreboard.ociweb.com/doc_master_tick_linux_gcc_d1o0/index.html

Reply:
Right, I agree.  This is similar to a discussion I had with @iguessthislldo on #1094.